pkg/dispatchserver/config.go

```diff
@@ -36,8 +36,11 @@ type serverConfig struct {
 	DisableHTTP       bool   `mapstructure:"disable-http" json:"disable-http"`
 	TLSPort           int    `mapstructure:"tls-port" json:"tls-port"`
 	EnableTLS         bool   `mapstructure:"enable-tls" json:"enable-tls"`
+	LetsEncrypt       bool   `mapstructure:"lets-encrypt" json:"lets-encrypt"`
+	Production        bool   `mapstructure:"production" json:"production"`
 	TLSCertificate    string `mapstructure:"tls-certificate" json:"tls-certificate"`
 	TLSCertificateKey string `mapstructure:"tls-certificate-key" json:"tls-certificate-key"`
+	Domain            string `mapstructure:"domain" json:"domain"`
```

Comment:
If the domain is needed, how about we merge lets-encrypt and this flag and have only one, `lets-encrypt-domain` flag? If it's empty, we disable lets encrypt. if it's set, we use the value as the domain.